node.gyp

```diff
@@ -577,7 +577,14 @@
         'OBJ_GEN_PATH': '<(OBJ_DIR)/node/gen',
         'OBJ_TRACING_PATH': '<(OBJ_DIR)/node/src/tracing',
         'OBJ_SUFFIX': 'o',
+        'OBJ_SEPARATOR': '/',
         'conditions': [
+          ['GENERATOR=="ninja"', {
+            'OBJ_PATH': '<(OBJ_DIR)/src/',
```

Comment:
don't end paths with a `/` it breaks on Windows (when it's the end of the final argument)